demo/sample_azure_iot_embedded_sdk_pkcs11/linux/Makefile

```diff
@@ -0,0 +1,221 @@
+CC := gcc
```

Comment:
Do we have a plan to cmake this at some point?  Or if we can't and we need such Makefile's floating around generally, is there some way we could set common "makefile.inc" (maybe a Windows build system term, but whatever Linux is) to have common set of logic to avoid tons of dup.

Reply:
We will move our sample linux based one to Cmake